### ISSUE TITLE
Boston, MA - Date: May 21 & 22 - Course: Building Web Apps With Go - Venue: Pivotal Labs

### ISSUE THREAD

**Issue by jboursiquot** [gobridge/workshops#18]
Organizer: @jboursiquot
Venue: Pivotal Labs (145 Broadway, Cambridge, MA 02142)
Food Sponsors: Pivotal Labs, Komand
Date: May 21st and 22nd, 2016
Curriculum: Building Web Apps with Go (https://github.com/gobridge/bwag)
### Logistic items
- [x] Has venue and it sits this many people: 35
- [x] Has sponsor for food and drinks: Partial -- Pivotal for Day 1 Lunch. __ for Day 2 Lunch.
- [x] Has organizer(s): @jboursiquot
- [x] Has teacher(s): @jboursiquot (others will be added once confirmed)
- [x] Has date: May 21 & 22 2016
- [x] [Course selected](https://github.com/gobridge/workshops/blob/master/available_courses.md#course-descriptions-and-corresponding-levels)
- [x] Draft of event started on BridgeTroll (https://www.bridgetroll.org/events/257)
- [x] Included in the event description if bikes are allowed inside or not (ask venue)
- [x] Included in the event description if child care is available or not

(Issue ready :dart: to be labeled as `scheduled` :boom::v::tada::balloon::rocket: after all tasks above are completed:exclamation:)
### A couple weeks before the event
- [x] Order placed for catering, if applicable
- [ ] Child care arranged for, if applicable
### First steps
- [x] Self-assigned the issue
- [x] Pinged the person who opened the issue
- [x] Google Hangout scheduled (if this is a first timer)

Continue taking the actions necessary to bring issue to a `scheduled` status according to our [issue workflow](https://github.com/gobridge/workshops/blob/master/issue_workflow.md).
### After event is labeled as `scheduled`
- [x] Bridge Troll event made public
- [x] Opened a meetup event to point to the Bridge Troll event (RSVP on meetup should be closed) (http://www.meetup.com/gobridge/events/230046573/)
- [x] Added a PR to add event to GoBridge's website (remove past events while at it)
- [x] First social media announcement's gone out
- [x] Conducted organizer training
- [x] Conducted teacher training
- [x] Assignee added event date -1 week to calendar as reminder
### Presentations
- [x] [Opening presentation](https://github.com/gobridge/workshop-material/blob/master/organizer-welcome.slide) forked, edited, and ready to present
- [x] [Closing presentation](https://github.com/gobridge/workshop-material/blob/master/organizer-closing.slide) forked, edited, and ready to present
### Event date -1 week
- [x] Confirmed venue and date
- [x] More rounds of announcements on social media. Mention how many spots left

**Comment by carlisia** [comment]
@jboursiquot how great! Would you please email me at carlisia@golangbridge.org so we can iron out details? Thanks!

**Comment by jboursiquot** [comment]
Great! Will do.

**Comment by carlisia** [comment]
Hi @jboursiquot, thanks for emailing me. I've added some new instructions that you might find useful: https://github.com/gobridge/workshops

**Comment by jboursiquot** [comment]
Update: 
- Venue's on lock
- Curriculum selected
- Food's tentatively sponsored, waiting on confirmation.
- Teachers and TAs coming together.

**Comment by carlisia** [comment]
Thanks for the update @jboursiquot, and great job! Let me know when you have a date and I'll add it to BridgeTroll so ppl can start signing up, including the volunteers.

**Comment by jboursiquot** [comment]
@carlisia update posted. Date is locked, Feb 6, 2016.

Going to reach out to some companies around town to see about getting child care and breakfast sponsored. Stretch goal, getting transportation sponsored as well.

**Comment by jboursiquot** [comment]
@carlisia, draft [BridgeTroll event](https://www.bridgetroll.org/events/237) in place. I set the Chapter as RailsBridge Boston for now but we'll probably want to change that--if we have a Boston chapter that is.

**Comment by carlisia** [comment]
@jboursiquot :+1: 

I'm working with @tjgrathwell and @lilliealbert from BridgeTroll to get the GoBridge Boston chapter inserted.

**Comment by carlisia** [comment]
@jboursiquot I just added a PR for BridgeTroll to have a listing of each of our courses: https://github.com/railsbridge/bridge_troll/pull/414. Once that is merged than you will need to select the Building Web Apps With Go.

Still waiting for the insertion of a GoBridge Boston Chapter, will let you know as soon as I hear back.

**Comment by lilliealbert** [comment]
@jboursiquot @carlisia Just added GoBridge Boston! Will hopefully soon have a UI for chapter creation so that these things don't have to be done via the Rails console :smile_cat:

**Comment by carlisia** [comment]
Wooohooo! Awesome @lilliealbert! Thank you! Did you also add London? We are ready to go with that one too! :boom:

**Comment by lilliealbert** [comment]
Alright, just added.

**Comment by carlisia** [comment]
:+1:

**Comment by carlisia** [comment]
@jboursiquot do you think the teachers would be available to do the google hangout this Saturday as well? I could cover the teachers training too. c/c @markbates, @timraymond (12/12 11AM EST.)

**Comment by carlisia** [comment]
@jboursiquot please figure out how many people we should cap this event at based on the venue and teachers' preference.

Once this PR https://github.com/railsbridge/bridge_troll/pull/414 is merged, you will be ready to publish the event. There is also a Boston GoBridge Chapter to select now.

**Comment by carlisia** [comment]
@jboursiquot that issue has been merged so you should be all set to open the Bridge Troll event!

**Comment by jboursiquot** [comment]
Hey @carlisia, I've submitted the BridgeTroll event for approval: https://www.bridgetroll.org/events/237.

I'm still on the lookout for childcare and afterparty sponsors but will update the event as they come.

**Comment by pdurbin** [comment]
Is this event still on? If not, can https://www.bridgetroll.org/events/237 please be updated?

**Comment by jboursiquot** [comment]
@pdurbin The event was supposed to be removed by now. I'll have to update the details to say it's postponed indefinitely until it can actually be removed from the site. Thanks for bringing this up.

**Comment by jboursiquot** [comment]
Update: Getting the ball rolling on this again. Met with a new venue sponsor on Friday (Pivotal Labs in Cambridge) and they're excited about hosting a GoBridge and have a well-suited space for meetups and workshops. I will select new date(s) soon and attempt to avoid any other major events that might be in conflict (like last time and the Super Bowl).

**Comment by jboursiquot** [comment]
Update: We're on! May 21, 22. Event's public on http://golangbridge.org and on BridgeTroll http://bit.ly/gobridge-boston.

@carlisia do you hold the power to tweet it from @golangbridge?

**Comment by carlisia** [comment]
@jboursiquot yes. Wanna give me the copy you want? Also, awesome!

**Comment by jboursiquot** [comment]
@carlisia 

```
GoBridge Boston is back! We’re holding our free two-day workshop "Building Web Apps with Go” on 5/21-22: bit.ly/gobridge-boston #golang
```

**Comment by carlisia** [comment]
Done: https://twitter.com/GolangBridge/status/715955242869919749

**Comment by jboursiquot** [comment]
Finally, food sponsors are on lock as of today and not a moment too soon :)

**Comment by jboursiquot** [comment]
This one's done.
